app/globals.css

```diff
@@ -3,13 +3,26 @@
 @tailwind utilities;
 
 :root {
-  --background: #ffffff;
-  --foreground: #171717;
+  --background: #f4f7f5;
+  --foreground: #f4f7f5;
+  --color-primary: #21a0a0;
+  --color-secondary: #046865;
+  --color-text: #074846;
+  --color-note: #e9f7ee;
+}
+
+.dark {
+  --background: #332d34;
+  --foreground: #f4f7f5;
+  --color-primary: #52154e;
+  --color-secondary: #310d2e;
+  --color-text: #efe9e7;
+  --color-note: #ecd5ec;
 }
```

Comment:
I don't love that I'm mapping two themes with names like "Tidepool" to a more standard "light-dark" - the theme could use more structure